spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -266,6 +346,48 @@ partition count of the topic in the case of Kafka.
 If neither a `partitionSelectorClass` nor a `partitionSelectorExpression` is
 present the result is `key.hashCode() % partitionCount`.
 
+
+==== Overriding application properties during stream deployment
+
+Application properties that are defined during deployment overrides the same properties defined during the stream creation.
+
+For example, the following stream has application properties defined during stream creation:
+
+[source,bash]
+----
+dataflow:> stream create --definition "time --fixed-delay=5 | log --level=WARN" --name ticktock
+----
+
+To override these application properties, one can specify the new property values during deployment:
+
+[source,bash]
+----
+dataflow:>stream deploy ticktock --properties "app.time.fixed-delay=4,app.log.level=ERROR"
+----
+
+
+=== Deployment properties
+
+When deploying the stream, properties that control the deployment of the apps into the target platform are known as `deployment` properties.
+For instance, one can specify how many number of instances need to be deployed for the specific application defined in the stream using the deployment property called `count`.
+
+==== Passing instance count as deployment property
+
+If you would like to have multiple instances of an application in the stream, you
+can include a property with the deploy command:
+
+[source,bash]
+----
+dataflow:> stream deploy --name ticktock --properties "app.time.count=3"
+----
+
+Note that `count` is the *reserved* property name used by the underlying deployer. Hence, if the application also has a custom property named `count`, it is *not* supported
+ when specified in `short-hand` form during stream `deployment` as it could conflict with the `instance` count deployer property. Instead, the `count` as a custom application property can be
```

Comment:
overuse of backticks here (should only be used for code, shell commands, property names, etc - not things like "deployment")